nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -20,26 +22,34 @@ define([
     Events,
     Jupyter,
     html,
+    CellTabs,
     CellControlPanel,
-    ConfigureWidget
+    ConfigureWidget,
+    CategoryPanel
 ) => {
     'use strict';
     const CELL_TYPE = 'app-bulk-import';
 
     const div = html.tag('div');
 
-    class DefaultWidget {
-        constructor() {
+    function DefaultWidget() {
```

Comment:
(not specific to this line of code!)

It looks like it's just this module that now uses the `class` paradigm -- do you want to convert it over to the factory pattern as part of this PR?

Reply:
I did. I wonder how that got reverted.
Too much branch switching today. Meh.

Reply:
I can do that in this PR if that's helpful. I don't want it to be too annoyingly huge to review.

Reply:
It's fine by me if it's a bit bigger. It shouldn't make too much difference.

Reply:
It's gonna turn out to be a lot bigger as most lines are touched. I'll set up a new PR specifically for that.